.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI_dev
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check current docker
+        run: docker version
+
+      - name: checkout
+        uses: actions/checkout@v1
+
+      - name: install go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
+
+      - name: install golangci-lint
+        run: |
+          go get github.com/golangci/golangci-lint/cmd/golangci-lint
+
+      - name: lint
+        run: |
+          export PATH=$PATH:$(go env GOPATH)/bin
+          ls -la
+          golangci-lint run ./...
+
+      - name: test
```

Comment:
why u run all jobs in one job? maybe need devide by lint, test, coverage?

Reply:
But i don't need a fresh instance of worker for each command and i don't want to spend active time on installing golang on each instance. So I don't think that it would be effective to use different jobs.